pygfx/renderers/wgpu/_flusher.py

```diff
@@ -206,11 +217,15 @@ def _create_pipeline(self, src_texture_view, dst_format):
                 for (var x:i32 = -support; x <= support; x = x + 1) {
                     let step = vec2<i32>(x, y);
                     let index = clamp(base_index + step, min_index, max_index);
-                    let dist = length(ref_index - vec2<f32>(index));
-                    let t = dist / sigma;
-                    let w = exp(-0.5 * t * t);
-                    val = val + textureLoad(r_color, index, 0) * w;
-                    weight = weight + w;
+                    //var n = textureNumSamples(r_color);
+                    for (var i=0; i<4; i=i+1) {
+                        let subindex = sample_positions4[i];
+                        let dist = length(ref_index - vec2<f32>(index) - 0.5 - subindex);
```

Comment:
In any case, that `-0.5` here should be applied regardless, to correctly center the targets.